src/graphql/typeDefs/mutations/asset.graphql

```diff
@@ -0,0 +1,4 @@
+type AssetMutation {
+  "Updates referenced asset"
+  update(asset: AssetUpdateInput!): Asset
```

Comment:
this should have a security role.

Reply:
messy, messy, messy. On it

Reply:
Ah, now I remember, not sure what to set it to. Member perhaps

Reply:
Nope, never mind. It has to be admin until some of the other security stuff is added